src/main/drivers/barometer/barometer_bmp280.c

```diff
@@ -39,6 +39,46 @@
 
 #if defined(USE_BARO) && (defined(USE_BARO_BMP280) || defined(USE_BARO_SPI_BMP280))
 
+
+#define BMP280_I2C_ADDR                      (0x76)
```

Comment:
This does not seem to be related to the BMP388. Can this go into a separate pull request?

Reply:
this was a case of fixing old code to match current coding standards as new code was introduced.  It can go into a new PR if you really want, but since we also have to change the method names of the bmp280 drivers as well as the bmp388 drivers I think it makes sense to do all the cleanups at the same time the new code is introduced.  it's already in a different cleanup commit with details - see 8795a71855539dc59ca2bb431d99385d0d460bb2.

Reply:
I think it makes sense and separate cleanup and new code into separate pull requests.

Reply:
traditionally in teams I've worked in cleanup changes go with the feature that caused the discovery of cleanup work, but cleanups go in separate commits so that if they break something they can be reverted.

the issue I have now is the time it takes to separate all the code changes from this PR into another one and making something is not lost or forgotten since the cleanups affect the new code too.

would *really* rather have related cleanups in this PR.  My time is very short for the next 2 weeks as I have house guests and other scheduled work.

Reply:
cleanup commits added to this PR.